CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
 
 ## [0.10.0] - 2023-07-05
 
+
+### Changed 
```

Comment:
'### Fixed' instead `Changed`

Something like:
```
### Fixed
- Fixed Class loading issues when running Http Connector from SQL Client on Flink 1.16
+ your description below.
```

Please try to format like other "Fixed"  items from the changelog ( - tag at the beginning)